app/views/static_pages/privacy_notice.html.erb

```diff
@@ -48,6 +48,7 @@
         your email address if you include it with feedback to us on the service or agree to be contacted for user research
         purposes
       </li>
+      <li>your Internet Protocol (IP) address</li>
```

Comment:
Do we need to say how long we store it for and why we need to store it?

Reply:
I am not sure, I felt like that was covered in the rest of the text?

Reply:
You're right, it does, I just re-read it. I wonder if we should be explicit that we collect IP addresses for security and not anything like analytics etc?

Reply:
I grabbed this from teaching vacancies but they mention it in the 'Who we will make your personal data available to' section.
`for security purposes: collecting part of the IP addresses of people visiting the Teaching Vacancies website (for example, if we were receiving continuous direct denial of service attacks from an IP address range then we could block it)`

Perhaps we should put something similar in the 'how we will use you information' section

Reply:
Note that we're not doing the same thing as they were. We collect full IP addresses, not partial, and we keep them forever.

Reply:
We plan to get the right team inside DfE to look at this, I've linked to the Trello card.